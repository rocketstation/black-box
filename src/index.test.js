var Fela = require('fela')
var React = require('react')
var ReactFela = require('react-fela')

const renderer = require('react-test-renderer')

const $ = require('./index.js')

test('', () => {
  expect(
    renderer
      .create(
        React.createElement(
          ReactFela.RendererProvider,
          { renderer: Fela.createRenderer() },
          $(
            'div',
            {
              className: 'test',
              onClick: () => {
                console.log('test')
              },
            },
            {
              backgroundColor: 'white',
              color: 'black',
            },
            $('div', null, { height: '10px' }),
            $('div', null, null, 'Hello'),
            $('div', null, null, 'World'),
            $('div', null, { height: '10px' })
          )
        )
      )
      .toJSON()
  ).toMatchSnapshot()
})
