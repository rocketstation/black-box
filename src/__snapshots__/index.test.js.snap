// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<div
  className="a b test"
  onClick={[Function]}
>
  <div
    className="c"
  />
  <div>
    Hello
  </div>
  <div>
    World
  </div>
  <div
    className="c"
  />
</div>
`;
